base/indices.jl

```diff
@@ -329,11 +329,15 @@ to_indices(A, I::Tuple{}) = ()
 to_indices(A, I::Tuple{Vararg{Int}}) = I
 to_indices(A, I::Tuple{Vararg{Integer}}) = (@inline; to_indices(A, (), I))
 to_indices(A, inds, ::Tuple{}) = ()
-to_indices(A, inds, I::Tuple{Any, Vararg{Any}}) =
-    (@inline; (to_index(A, I[1]), to_indices(A, _maybetail(inds), tail(I))...))
+function to_indices(A, inds, I::Tuple{Any, Vararg{Any}})
+    @inline
+    head = _to_indices1(A, inds, I[1])
+    rest = to_indices(A, _cutdim(inds, I[1]), tail(I))
```

Comment:
Is this `_cutdim` supposed to be overloaded? Otherwise I think we can just make it 1-arg method.